firefox_puppeteer/docs/index.rst

```diff
@@ -32,15 +32,10 @@ Libraries
 The following libraries are currently implemented. More will be added in the
 future. Each library is available from an instance of the FirefoxTestCase class.
 
-.. autoclass:: Puppeteer
-   :members:
```

Comment:
We only expose most-used API modules here. We don't need any ui module on that class.